### PR TITLE
feat: update tutorial

### PR DESCRIPTION
**Pull request by alexcodelf**

### CODE REVIEW

File: docs/tutorials/how-to-create-template.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ sidebar_position: 4
 
 # How to Create a Template
```
Comment (gitlawr) [inline docs/tutorials/how-to-create-template.md:5]: What about rename the title to `Create/Write a Template`

File: docs/tutorials/how-to-create-template.md [hunk 2]
```diff
@@ -14,11 +14,11 @@ This tutorial explains how to create a Alibaba Cloud EC2 template and use it to
 ## Create a repository on GitHub
 
 1. Create a new repository on GitHub of your own.
- Here we use the repository [demo](https://github.com/walrus-catalog-demo/demo)
+ Here we use the repository [terraform-alicloud-rds-mysql](https://github.com/walrus-catalog/terraform-alicloud-rds-mysql)
```
Comment (gitlawr) [inline docs/tutorials/how-to-create-template.md:17]: There's no demo directory in the repo. 
Why users need to create files if all the files are present after the clone?

Comment (alexcodelf) [reply docs/tutorials/how-to-create-template.md:17]: This is an example to show files should be created for a new template if users want to create their own template.

We may add introduction here, like `Your can replace the git with your own and create the following files or using our repo to modify as your template`?

File: docs/tutorials/how-to-create-template.md (hunk 1 above)
Comment (alexcodelf) [reply docs/tutorials/how-to-create-template.md]: change to `Create your first template`

File: docs/tutorials/how-to-create-template.md (hunk 2 above)
Comment (alexcodelf) [reply docs/tutorials/how-to-create-template.md]: Updated, using a simple kubernetes template to help user quickly create their template.

**Comment by alexcodelf** [comment]
Updated. Write a simple document introducing how to create templates.

**Review by gitlawr — APPROVED** [review]
LGTM

**Pull request merged**